packages/nextjs/components/example-ui/ContractInteraction.tsx

```diff
@@ -66,7 +66,7 @@ export const ContractInteraction = () => {
                   className={`btn btn-primary rounded-full capitalize font-normal font-white w-24 flex items-center gap-1 hover:gap-2 transition-all tracking-widest ${
                     isLoading ? "loading" : ""
                   }`}
-                  onClick={writeAsync}
+                  onClick={() => writeAsync()}
```

Comment:
We wont be able to `onClick={writeAsync}` anymore 😅 but I think its ok

Reply:
I don't see an issue either, the behavior will be even more aligned with wagmi's `writeAsync`